cmd/compose/pull.go

```diff
@@ -86,6 +87,12 @@ func runPull(ctx context.Context, backend api.Service, opts pullOptions, service
 		project.Services = enabled
 	}
 
+	for i, service := range project.Services {
+		if service.PullPolicy == "" {
+			project.Services[i].PullPolicy = types.PullPolicyAlways
+		}
+	}
+
```

Comment:
Explicitly default to `always` pull policy when not set for `pull` command. This seems to match user expected behavior but I don't think this new defaulting logic is documented or specified anywhere. Thoughts?